CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### 3.2.3 (2020-12-09)
+
+#### Notable Changes
+
+- [refactor: update gcs buckets](https://github.com/twreporter/plate/pull/202)
+
+#### Commits
+
+* [[`f03e847930`](https://github.com/twreporter/keystone-plate/commit/f03e847930)] - Merge pull request #202 from taylrj/fix-audio-path (Tai-Jiun Fang)
```

Comment:
The link invalid, it should be `https://github.com/twreporter/plate/commit/f03e847930`.
Following bullets also suffer from the typo.

Reply:
Thanks for the reminder, I created [PR](https://github.com/twreporter/plate/pull/206) to fix it.